app/src/main/AndroidManifest.xml

```diff
@@ -132,7 +130,6 @@
         <activity
             android:name=".ui.LaunchActivity"
             android:exported="true"
-            android:noHistory="true"
```

Comment:
Because we needed LaunchActivity to get the response of the LockActivity then launch the MainActivity, as it was done direclty by the LockActivity before.
It will be simplified by passing the class as parameter to the LockActvity now.